.prettierrc

```diff
@@ -1,4 +1,14 @@
 {
-  "printWidth": 180,
-  "singleQuote": true
+	"printWidth": 120,
+	"quoteProps": "consistent",
+	"singleQuote": true,
+	"trailingComma": "es5",
+	"useTabs": true,
```

Comment:
I presume you are going to have a lot of whitespace updates in your commits going forward with this. Do you want that?

Reply:
Hmmm 🤔 I'll look into:
- Creating a separate PR running prettier on the entire app
and/or
- Investigating if it's possible to run prettier and not effect the git history

Reply:
Usually the strategy is to add (mass) prettier/lint commits in .git-blame-ignore-revs: https://github.com/HarperDB/harperdb/blob/main/.git-blame-ignore-revs
I just didn't know if you really wanted to do that right now.